requirements.txt

```diff
@@ -13,8 +13,11 @@ numpy>=1.15.0
 requests>=2.13.0,<3.0.0
 plac>=0.9.6,<1.2.0
 tqdm>=4.38.0,<5.0.0
-importlib_metadata>=0.20; python_version < "3.8"
 pydantic>=1.3.0,<2.0.0
+# Official Python utilities
+setuptools
```

Comment:
I guess we really do have to leave these unpinned upwards, but should we set a minimum lower version? I don't know what sort of ancient setuptools someone might have.